package.json

```diff
@@ -30,6 +30,7 @@
     "@riderize/passport-strava-oauth2": "^1.1.1",
     "@simplewebauthn/server": "^5.3.0",
     "@types/passport-oauth2": "^1.4.11",
+    "@workos-inc/node": "^2.12.0",
```

Comment:
It seems we're not using this package

Reply:
true, it's included as a dependency in `passport-workos`

Reply:
actually no, we need to install it so `passport-workos` works